### PR TITLE
Add new volunteer Ashraf chowdury

### PR DESCRIPTION
**Pull request by ashrafchowdury**

### CODE REVIEW

**Comment by codewithshubhi** [comment]
Thanks for raising PR and contributing @ashrafchowdury :)

File: data/users/ashrafchowdury.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "name": "Ashraf Chowdury",
+  "bio": "Font-End developer",
```
Comment (codewithvoid) [inline data/users/ashrafchowdury.json:3]: nit: Fontend developer

File: data/users/ashrafchowdury.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  "image": "https://avatars.githubusercontent.com/u/87828904?v=4",
+  "categories": [
+    {
+      "slug": "web-development",
+      "topics": ["JavaScript", "React.js"]
```
Comment (codewithvoid) [inline data/users/ashrafchowdury.json:8]: ReactJS

**Comment by codewithvoid** [comment]
Looks good @ashrafchowdury . Added small comments.

**Comment by ashrafchowdury** [comment]
Resolve all **issues**

**Comment by codewithvoid** [comment]
Thank you!

**Review by codewithvoid — APPROVED** [review]

**Pull request merged**